torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -789,10 +807,22 @@ def _generate_comparision_values(self, input_info: Dict, weight_info: Dict) -> D
             weight_ratio = self._calculate_range_ratio(weight_info[module_fqn], self.WEIGHT_STR, module_fqn)
             input_ratio = self._calculate_range_ratio(input_info[module_fqn], self.INPUT_STR, module_fqn)
 
+            # if mismatched size, because of grouping, we want to replicate weight enough times
+            weight_channels = len(weight_ratio)
+            input_channels = len(input_ratio)
+            if weight_channels != input_channels:
```

Comment:
I'm wondering if we should be checking that the module is Conv2d here, otherwise it might be accepting cases that we do not want to handle

Reply:
I think this mitigated by the fact that above, only linears are convs are listed as supported modules, so these are the only modules we could even be looking at in this case, so i think we're safe here.